Dockerfile

```diff
@@ -8,9 +7,15 @@ ENV NODE_ENV 'production'
 ENV ZEN_ENV 'br'
 ENV ZEN_PROD 1
 ENV ZEN_DOCKER 1
-ENV DATABASE_URL 'postgresql://test_admin:zenpass@zen_postgres/zenysis'
 ENV PYTHONPATH "${ZENYSIS_SRC_ROOT}:${PYTHONPATH}"
 
+RUN mkdir -p ${INSTALL_PATH}
+RUN mkdir -p /data/output
+
+WORKDIR ${INSTALL_PATH}
+
+ADD . .
```

Comment:
I'm not a big fan of adding all files from the root of the source directory into the dockerfile, since it likely will include a lot of things that are not needed to run.

To make this a little easier, I have published our latest docker files with commit 646794b16235